store/src/db/mod.rs

```diff
@@ -211,8 +227,8 @@ impl Db {
             .get()
             .await?
             .interact(move |conn| -> anyhow::Result<()> {
-                let span = span!(Level::INFO, COMPONENT, "writing new block data to DB");
-                let guard = span.enter();
+                let _span =
+                    info_span!(target: COMPONENT, "store:db:apply_block:Writing new block data to database").entered();
```

Comment:
Similar comment to the one above: instead of manually prefixing this with `store::db::apply_block`, let's add a `TODO` here and then also create an issue (probably this could be the same issue as mentioned in the previous comment).

Also, I would change the span name to "write_block_to_db".